pkg/test/ginkgo/external.go

```diff
@@ -83,9 +84,17 @@ func extractBinaryFromReleaseImage(tag, binary string) (string, error) {
 		return "", fmt.Errorf("cannot determine release image from ClusterVersion resource")
 	}
 
-	if err := runImageExtract(releaseImage, "/release-manifests/image-references", tmpDir, ""); err != nil {
-		return "", fmt.Errorf("failed extracting image-references: %w", err)
+	err = runImageExtract(releaseImage, "/release-manifests/image-references", tmpDir, "")
+	if err != nil {
+		// Retry with the RELEASE_IMAGE_LATEST environment variable, if specified
+		if releaseImage = os.Getenv("RELEASE_IMAGE_LATEST"); releaseImage != "" {
```

Comment:
Do we ever test with RELEASE_IMAGE_INITIAL? Maybe it's better to use a different environment variable (like RELEASE_IMAGE_OVERRIDE) and specify that in all the disconnected jobs so we have more control over what's used.